src/scripts/download-new-songs.ts

```diff
@@ -162,6 +173,7 @@ const ffmpegOpusJob = async (mp3File: string) => {
     let oggPartWithPath = `${oggFileWithPath}.part`;
     let oggFfmpegOutputStream = fs.createWriteStream(oggPartWithPath);
 
+    logger.info(`Starting ffmpeg process for ${mp3File}`)
```

Comment:
use more descriptive wording, we're encoding to ogg, mentioning ffmpeg doesn't tell us much